test/instrumentation/documentation/main.go

```diff
@@ -60,85 +60,58 @@ components using an HTTP scrape, and fetch the current metrics data in Prometheu
 
 Stable metrics observe strict API contracts and no labels can be added or removed from stable metrics during their lifetime.
 
-<table class="table metrics" caption="This is the list of STABLE metrics emitted from core Kubernetes components">
-<thead>
-	<tr>
-		<th class="metric_name">Name</th>
-		<th class="metric_stability_level">Stability Level</th>
-		<th class="metric_type">Type</th>
-		<th class="metric_help">Help</th>
-		<th class="metric_labels">Labels</th>
-		<th class="metric_const_labels">Const Labels</th>
-		<th class="metric_deprecated_version">Deprecated Version</th>
-	</tr>
-</thead>
-<tbody>
-{{range $index, $metric := .StableMetrics}}
-<tr class="metric"><td class="metric_name">{{with $metric}}{{.BuildFQName}}{{end}}</td>
-<td class="metric_stability_level" data-stability="{{$metric.StabilityLevel | ToLower}}">{{$metric.StabilityLevel}}</td>
-<td class="metric_type" data-type="{{$metric.Type | ToLower}}">{{$metric.Type}}</td>
-<td class="metric_description">{{$metric.Help}}</td>
-{{if not $metric.Labels }}<td class="metric_labels_varying"></td>{{else }}<td class="metric_labels_varying">{{range $label := $metric.Labels}}<div class="metric_label">{{$label}}</div>{{end}}</td>{{end}}
-{{if not $metric.ConstLabels }}<td class="metric_labels_constant"></td>{{else }}<td class="metric_labels_constant">{{range $key, $value := $metric.ConstLabels}}<div class="metric_label">{{$key}}:{{$value}}</div>{{end}}</td>{{end}}
-{{if not $metric.DeprecatedVersion }}<td class="metric_deprecated_version"></td>{{else }}<td class="metric_deprecated_version">{{$metric.DeprecatedVersion}}</td>{{end}}</tr>{{end}}
-</tbody>
-</table>
+<div class="metrics">
+{{- range $index, $metric := .StableMetrics -}}
+	<div class="metrics-wrapper">
+	<div class="metric_name">{{with $metric}}{{.BuildFQName}}{{end}}</div>
+	<div>{{$metric.Help}}</div>
+	<ul>
+	<li data-stability="{{$metric.StabilityLevel | ToLower}}"><span class="metric-data">Stability Level:</span>{{$metric.StabilityLevel}}</li>
+	<li data-type="{{$metric.Type | ToLower}}"><span class="metric-data">Type:</span>{{$metric.Type}}</li>
+	{{if not $metric.Labels }}<div class="metric_labels_varying"></div>{{else }}<li class="metric_labels_varying"><span class="metric-data">Labels:</span>{{range $label := $metric.Labels}}<span class="separate-labels">{{$label}}</span>{{end}}</li>{{end}}
+	{{if not $metric.ConstLabels }}<div class="metric_labels_constant"></div>{{else }}<li class="metric_labels_constant"><span class="metric-data">Const Labels:</span>{{range $key, $value := $metric.ConstLabels}}<span class="metric_label">{{$key}}:{{$value}}</span>{{end}}</li>{{end}}
+	{{if not $metric.DeprecatedVersion }}<div class="metric_deprecated_version"></div>{{else }}<li class="metric_deprecated_version"><span class="metric-data">Deprecated Versions:</span>{{$metric.DeprecatedVersion}}</li>{{end}}
+	</ul>
+	</div>
```

Comment:
(untested)
```suggestion
	<div class="metric stable-metric" data-stability="{{$metric.StabilityLevel | ToLower}}">
	<div class="metric_name">{{with $metric}}{{.BuildFQName}}{{end}}</div>
	<div class="metric_help">{{$metric.Help}}</div>
	<ul>
	<li><label class="metric_detail">Stability Level:</label><span class="metric_stability_level">{{$metric.StabilityLevel}}</span></li>
	<li data-type="{{$metric.Type | ToLower}}"><label class="metric_detail">Type</label> <span class="metric_type">{{$metric.Type}}</span></li>
	{{if $metric.Labels }}<li class="metric_labels_varying"><label class="metric_detail">Labels</span>{{range $label := $metric.Labels}} <span class="metric_label">{{$label}}</span>{{end}}</li>{{end}}
	{{if $metric.ConstLabels }}<li class="metric_labels_constant"><label class="metric_detail">Const Labels</label> {{range $key, $value := $metric.ConstLabels}} <span class="metric_label">{{$key}}:{{$value}}</span>{{end}}</li>{{end}}
	{{if $metric.DeprecatedVersion }}<li class="metric_deprecated_version"><label class="metric_detail">Deprecated Versions</label> <span>{{$metric.DeprecatedVersion}}</span></li>{{end}}
	</ul>
	</div>
```

Does this work?

Reply:
I made some amendments and tested it, The code works. I'll be pushing it over the latest commit

Reply:
I've pushed the suggested amendments, The wrapper is indeed better now @sftim